artifactory_test.go

```diff
@@ -4995,11 +4995,12 @@ func TestArtifactoryReplicationCreate(t *testing.T) {
 	assert.NoError(t, err)
 
 	// Init tmp dir
-	specFile, err := tests.CreateSpec(tests.ReplicationTempCreate)
+	templatePath := filepath.Join(tests.GetTestResourcesPath(), tests.ReplicationTempCreate)
+	templatePath, err = tests.ReplaceTemplateVariables(templatePath, "")
```

Comment:
Let's add a description for the `ReplaceTemplateVariables` function (noy here, but in the function itself).